<?xml version="1.0" encoding="UTF-8"?>
<!-- ===============================================
  Copyright (C) 2014 ADVENTURE Project
  All Rights Reserved
================================================ -->
<ui version="4.0">
    <class>MainWindow</class>
    <widget class="QMainWindow" name="MainWindow">
        <property name="geometry">
            <rect>
                <x>0</x>
                <y>0</y>
                <width>600</width>
                <height>400</height>
            </rect>
        </property>
        <property name="windowTitle">
            <string>MapEditor for ADVENTURE_Mates</string>
        </property>
        <property name="windowIcon">
            <iconset resource="map_editor.qrc">
            <normaloff>:/icon/map_editor.png</normaloff>:/icon/map_editor.png</iconset>
        </property>
        <widget class="QWidget" name="centralWidget"/>
        <widget class="QMenuBar" name="menuBar">
            <property name="geometry">
                <rect>
                    <x>0</x>
                    <y>0</y>
                    <width>600</width>
                    <height>19</height>
                </rect>
            </property>
            <widget class="QMenu" name="menu_F">
                <property name="title">
                    <string>File(&amp;F)</string>
                </property>
                <addaction name="actionFileNew"/>
                <addaction name="actionFileOpen"/>
                <addaction name="separator"/>
                <addaction name="actionFileSave"/>
                <addaction name="actionFileSaveAs"/>
                <addaction name="separator"/>
                <addaction name="actionFileOption"/>
                <addaction name="separator"/>
                <addaction name="actionFileTestData"/>
                <addaction name="separator"/>
                <addaction name="actionExit"/>
                <addaction name="separator"/>
            </widget>
            <widget class="QMenu" name="menu_H">
                <property name="title">
                    <string>Help(&amp;H)</string>
                </property>
                <addaction name="actionAbout"/>
            </widget>
            <widget class="QMenu" name="menu_E">
                <property name="title">
                    <string>Edit(&amp;E)</string>
                </property>
                <addaction name="actionSelect"/>
                <addaction name="actionLarge"/>
                <addaction name="separator"/>
                <addaction name="actionIntsec"/>
                <addaction name="separator"/>
                <addaction name="actionRoad"/>
                <addaction name="actionRoadRepl"/>
                <addaction name="separator"/>
                <addaction name="actionLane2"/>
                <addaction name="actionLane4"/>
                <addaction name="separator"/>
                <addaction name="actionZoomIn"/>
                <addaction name="actionZoomOut"/>
                <addaction name="separator"/>
                <addaction name="actionDirJump"/>
                <addaction name="separator"/>
                <addaction name="actionConfigEditor"/>
            </widget>
            <widget class="QMenu" name="menu_I">
                <property name="title">
                    <string>Info(&amp;I)</string>
                </property>
                <addaction name="actionAllInfo"/>
            </widget>
            <addaction name="menu_F"/>
            <addaction name="menu_E"/>
            <addaction name="menu_I"/>
            <addaction name="menu_H"/>
        </widget>
        <widget class="QToolBar" name="mainToolBar">
            <attribute name="toolBarArea">
                <enum>TopToolBarArea</enum>
            </attribute>
            <attribute name="toolBarBreak">
                <bool>false</bool>
            </attribute>
            <addaction name="actionSelect"/>
            <addaction name="actionLarge"/>
            <addaction name="separator"/>
            <addaction name="actionIntsec"/>
            <addaction name="separator"/>
            <addaction name="actionRoad"/>
            <addaction name="actionRoadRepl"/>
            <addaction name="separator"/>
            <addaction name="separator"/>
            <addaction name="actionLane2"/>
            <addaction name="actionLane4"/>
            <addaction name="separator"/>
            <addaction name="actionZoomIn"/>
            <addaction name="actionZoomOut"/>
            <addaction name="separator"/>
            <addaction name="actionConfigEditor"/>
        </widget>
        <widget class="QStatusBar" name="statusBar"/>
        <action name="actionFileNew">
            <property name="text">
                <string>New(&amp;N)</string>
            </property>
            <property name="toolTip">
                <string>New</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+N</string>
            </property>
        </action>
        <action name="actionFileOpen">
            <property name="text">
                <string>Open(&amp;O)</string>
            </property>
            <property name="toolTip">
                <string>Open</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+O</string>
            </property>
        </action>
        <action name="actionFileSave">
            <property name="text">
                <string>Save(&amp;S)</string>
            </property>
            <property name="toolTip">
                <string>Save</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+S</string>
            </property>
        </action>
        <action name="actionFileSaveAs">
            <property name="text">
                <string>Save as(&amp;A)</string>
            </property>
            <property name="toolTip">
                <string>Save as</string>
            </property>
        </action>
        <action name="actionFileOption">
            <property name="text">
                <string>Setting(&amp;O)</string>
            </property>
            <property name="toolTip">
                <string>Setting</string>
            </property>
            <property name="shortcut">
                <string>Alt+O</string>
            </property>
        </action>
        <action name="actionExit">
            <property name="text">
                <string>Quit(&amp;Q)</string>
            </property>
            <property name="toolTip">
                <string>Quit</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+Q</string>
            </property>
        </action>
        <action name="actionAbout">
            <property name="text">
                <string>About MapEditor(&amp;A)</string>
            </property>
            <property name="toolTip">
                <string>About MapEditor</string>
            </property>
        </action>
        <action name="actionIntsec">
            <property name="checkable">
                <bool>true</bool>
            </property>
            <property name="icon">
                <iconset resource="map_editor.qrc">
                <normaloff>:/icon/tool_intsec.png</normaloff>:/icon/tool_intsec.png</iconset>
            </property>
            <property name="text">
                <string>Intersection(&amp;I)</string>
            </property>
            <property name="toolTip">
                <string>Edit Intersection</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+I</string>
            </property>
        </action>
        <action name="actionRoad">
            <property name="checkable">
                <bool>true</bool>
            </property>
            <property name="icon">
                <iconset resource="map_editor.qrc">
                <normaloff>:/icon/tool_road.png</normaloff>:/icon/tool_road.png</iconset>
            </property>
            <property name="text">
                <string>Section(&amp;R)</string>
            </property>
            <property name="toolTip">
                <string>Edit Section</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+R</string>
            </property>
        </action>
        <action name="actionRoadRepl">
            <property name="checkable">
                <bool>true</bool>
            </property>
            <property name="icon">
                <iconset resource="map_editor.qrc">
                <normaloff>:/icon/tool_road_repl.png</normaloff>:/icon/tool_road_repl.png</iconset>
            </property>
            <property name="text">
                <string>Replace section(&amp;P)</string>
            </property>
            <property name="toolTip">
                <string>Replace section</string>
            </property>
            <property name="shortcut">
                <string>Ctrl+P</string>
            </property>
        </action>
        <action name="actionZoomIn">
            <property name="icon">
                <iconset resource="map_editor.qrc">
                <normaloff>:/icon/tool_zoom_in.png</normaloff>:/icon/tool_zoom_in.png</iconset>
            </property>
            <property name="text">
                <string>Zoom in(&amp;Z)</string>
            </property>
            <property name="toolTip">
                <string>Zoom in</string>
            </property>
            <property name="shortcut">
                <string>+</string>
            </property>
        </action>
        <action name="actionZoomOut">
            <property name="icon">
                <iconset resource="map_editor.qrc">
                <normaloff>:/icon/tool_zoom_out.png</normaloff>:/icon/tool_zoom_out.png</iconset>
            </property>
            <property name="text">
                <string>Zoom out(&amp;O)</string>
            </property>
            <property name="toolTip">
                <string>Zoom out</string>
            </property>
            <property name="shortcut">
                <string>-</string>
            </property>
        </action>
        <action name="actionLane2">
            <property name="checkable">
                <bool>true</bool>
            </property>
            <property name="icon">
                <iconset resource="map_editor.qrc">
                <normaloff>:/icon/line2.png</normaloff>:/icon/line2.png</iconset>
            </property>
            <property name="text">
                <string>Section[1 lane each way](&amp;D)</string>
            </property>
            <property name="toolTip">
                <string>Create section[1 lanes each way]</string>
            </property>
        </action>
        <action name="actionLane4">
            <property name="checkable">
                <bool>true</bool>
            </property>
            <property name="icon">
                <iconset resource="map_editor.qrc">
                <normaloff>:/icon/line4.png</normaloff>:/icon/line4.png</iconset>
            </property>
            <property name="text">
                <string>Section[2 lanes each way](&amp;F)</string>
            </property>
            <property name="toolTip">
                <string>Create section[2 lanes each way]</string>
            </property>
        </action>
        <action name="actionFileTestData">
            <property name="text">
                <string>Test data(&amp;T)</string>
            </property>
            <property name="toolTip">
                <string>Test data</string>
            </property>
        </action>
        <action name="actionSelect">
            <property name="checkable">
                <bool>true</bool>
            </property>
            <property name="icon">
                <iconset resource="map_editor.qrc">
                <normaloff>:/icon/tool_select.png</normaloff>:/icon/tool_select.png</iconset>
            </property>
            <property name="text">
                <string>Select(S)</string>
            </property>
            <property name="toolTip">
                <string>Select</string>
            </property>
        </action>
        <action name="actionLarge">
            <property name="checkable">
                <bool>true</bool>
            </property>
            <property name="icon">
                <iconset resource="map_editor.qrc">
                <normaloff>:/icon/tool_large.png</normaloff>:/icon/tool_large.png</iconset>
            </property>
            <property name="text">
                <string>Simple view(&amp;L)</string>
            </property>
            <property name="toolTip">
                <string>Simple view</string>
            </property>
        </action>
        <action name="actionDirJump">
            <property name="text">
                <string>Jump to intersection(&amp;G)</string>
            </property>
            <property name="toolTip">
                <string>Jump to intersection</string>
            </property>
        </action>
        <action name="actionAllInfo">
            <property name="text">
                <string>Roadmap info(&amp;I)</string>
            </property>
            <property name="toolTip">
                <string>Roadmap info</string>
            </property>
        </action>
    </widget>
    <layoutdefault spacing="6" margin="11"/>
    <resources>
        <include location="map_editor.qrc"/>
    </resources>
    <connections/>
</ui>
